sdk/core/azure-mgmt-core/README.rst

```diff
@@ -0,0 +1,531 @@
+AutoRest: Python Client Runtime - Azure Module
```

Comment:
Haven't we switched all READMEs to markdown?

Reply:
Fixed